app/views/teacher_interface/registration_number/edit_ghana.html.erb

```diff
@@ -0,0 +1,37 @@
+<% content_for :page_title, title_with_error_prefix(t("application_form.tasks.items.registration_number.ghana"), error: @form.errors.any?) %>
+<% content_for :back_link_url, back_history_path(default: teacher_interface_application_form_path) %>
+
+<%= form_with model: @form, url: %i[registration_number teacher_interface application_form] do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <%= f.govuk_text_field :registration_number, label: { size: "l", tag: "h1" } %>
+
+  <% if @application_form.region.status_check_online? %>
+    <% if @application_form.region.status_information.present? %>
+      <%= raw GovukMarkdown.render(@application_form.region.status_information) %>
+    <% end %>
+
+    <% if @application_form.region.country.status_information.present? %>
+      <%= raw GovukMarkdown.render(@application_form.region.country.status_information) %>
+    <% end %>
+  <% end %>
+
+  <% if @application_form.region.sanction_check_online? %>
+    <% if @application_form.region.sanction_information.present? %>
+      <%= raw GovukMarkdown.render(@application_form.region.sanction_information) %>
+    <% end %>
+
+    <% if @application_form.region.country.sanction_information.present? %>
+      <%= raw GovukMarkdown.render(@application_form.region.country.sanction_information) %>
+    <% end %>
+  <% end %>
+
+  <%= govuk_details(summary_text: "When we might need more information") do %>
```

Comment:
I'm wondering if this section is still necessary for the Ghana view. I didn't see it in the screenshot. Do we need to keep it here?